superset-frontend/src/SqlLab/components/KeyboardShortcutButton/index.tsx

```diff
@@ -51,6 +53,8 @@ export const KEY_MAP = {
   [KeyboardShortcut.CtrlT]: userOS !== 'Windows' ? t('New tab') : undefined,
   [KeyboardShortcut.CtrlP]: t('Previous Line'),
   [KeyboardShortcut.CtrlShiftF]: t('Format SQL'),
+  [KeyboardShortcut.CtrlLeft]: t('Switch to the previous tab'),
+  [KeyboardShortcut.CtrlRight]: t('Switch to the next tab'),
```

Comment:
Not needed for this PR, but do you think it's worth adding a section somewhere on the docs site about keyboard shortcuts?

Reply:
FYI: The key mapping on this section has been implemented to be automatically added to the Keyboard Shortcut menu.
![Screenshot 2024-09-06 at 10 24 31 AM](https://github.com/user-attachments/assets/fc7e61d4-0a69-4843-9d11-998671b9975f)